tensordict/tensordict.py

```diff
@@ -1477,6 +1477,84 @@ def apply(
             out.lock_()
         return out
 
+    def map(
+        self,
+        fn: Callable,
+        dim: int = 0,
+        num_workers: int = None,
+        chunksize: int = None,
+        pool: mp.Pool = None,
+    ):
+        """Maps a function to splits of the tensordict across one dimension.
+
+        This method will apply a function to a tensordict instance by chunking
+        it in tensordicts of equal size and dispatching the operations over the
+        desired number of workers.
+
+        The function signature should be ``Callabe[[TensorDict], Union[TensorDict, Tensor]]``.
```

Comment:
why not writing the type hint then?

Reply:
we can do that too! Good point

I think that docstrings >> type hint as people should look at the doc rather than the code to see what a method is made for and how to use it.
(as in: I don't destroy my microwave to see how to use it but I read the manual :) )
But you're right, we can do both